old-web-redirector/src/Redirect.vue

```diff
@@ -0,0 +1,163 @@
+  computed: {
+    ...mapGetters(['user'])
+  },
+
+  async mounted() {
+    /*
+
+    index.php/apps/files/?dir=/&view=favorites -> favorites
+    index.php/apps/files/?dir=/&view=sharingin -> shared with me
+    index.php/apps/files/?dir=/&view=sharingout -> shared by others
+    index.php/apps/files/?dir=/&view=sharinglinks - shared by link
+    index.php/apps/files/?dir=/&view=projectspaces-personal -> projects
+    index.php/apps/files/?dir=/&view=eostrashbin -> trasbin
+
+
+    index.php/apps/files/?dir=/my/path& -> /eos/user/[letter]/[username]/my/path
+
+
+    index.php/apps/files?dir=/__myshares/mysharename%20(id%3A4235)/and/path -> [share path]/and/path
+    index.php/apps/files?dir=/__myprojects/myproject/and/path -> /eos/project/[letter]/[project]/and/path
+
+    index.php/s/ASFIAHSUAHS?path=%2Fand%2Fpath -> [token]/and/path
+
+    */
+
+    const isPublic = this.$route.name.includes('public')
+    const view = this.$route.query.view
+    const dir = this.$route.query.dir
+    const first = dir?.split('/').filter(Boolean)[0]
+
+    if (isPublic) {
+      console.log('Redirecting old url to public link')
+      this.$router.push({
+        name: 'files-public-files',
+        params: {
+          item: `${this.$route.params.token}${this.$route.query.path || ''}`
+        }
+      })
+    } else {
+      switch (view) {
+        case 'favorites':
+          console.log('Redirecting old url to favorites')
+          this.$router.push({
+            name: 'files-common-favorites'
+          })
+          break
+
+        case 'sharingin':
+          console.log('Redirecting old url to shared with me')
+          this.$router.push({
+            name: 'files-shares-with-me'
+          })
+          break
+
+        case 'sharingout':
+          console.log('Redirecting old url to shared by me')
+          this.$router.push({
+            name: 'files-shares-with-others'
+          })
+          break
+
+        case 'sharinglinks':
+          console.log('Redirecting old url to shared by link')
+          this.$router.push({
+            name: 'files-shares-via-link'
+          })
+          break
+
+        case 'projectspaces-personal':
+          console.log('Redirecting old url to projects view')
+          // TODO change after moving to spaces
+          this.$router.push({
+            name: 'files-spaces-project'
+          })
+          break
+
+        case 'eostrashbin':
+          console.log('Redirecting old url to trashbin')
+          this.$router.push({
+            name: 'files-trash-personal'
+          })
+          break
+
+        default:
+          switch (first) {
+            case '__myshares': {
+              console.log('Redirecting old url to a share')
+              const path = await this.getSharePath(dir, this.$client)
+              this.$router.push({
+                name: path ? 'files-spaces-personal' : 'files-shares-with-me',
+                params: {
+                  storageId: this.user.id,
```

Comment:
`storageId: 'personal'` when [pr cleaning urls ](https://github.com/cernbox/web/pull/135) gets accepted

Reply:
Yes, I think I added a "TODO" in the code for that